source/usage-examples/command.txt

```diff
@@ -0,0 +1,46 @@
+=============
+Run a Command
+=============
+
+.. default-domain:: mongodb
+
+You can run all raw database operations using the
+``MongoDatabase.runCommand()`` method. A raw database operation is a
+command you can execute directly on the MongoDB server CLI. Commands
+include administrative and diagnostic tasks, such as fetching server
+stats or initializing a replica set. Call the ``runCommand()`` method
+with a Bson command object on an instance of a database to run your raw
+database operation.
+
+.. note::
+    Use the :manual:`mongo shell </reference/mongo-shell>` for
+    administrative tasks instead of the Java driver whenever possible.
+
+The ``runCommand()`` method accepts a command in Bson format.
+By default, ``runCommand`` returns an object of type
+``org.bson.Document`` containing the output of the database command. You
+can specify a return type for ``runCommand()`` as an optional second
+parameter.
+
+Example
+-------
+
+.. include:: /includes/connect-guide-note.rst
+
+.. literalinclude:: /includes/usage-examples/code-snippets/Command.java
+  :language: java
+
+
+When you run the above command, you should see output similar to the
+following:
+
+.. code-block:: none
+
+   dbStats: {"db": "sample_mflix", "collections": 5, "views": 0, "objects": 75595, "avgObjSize": 692.1003770090614, "dataSize": 52319328, "storageSize": 29831168, "numExtents": 0, "indexes": 9, "indexSize": 14430208, "fileSize": 0, "nsSizeMB": 0, "ok": 1}
+
+For additional information on the classes and methods mentioned on this
+page, see the following API documentation:
+
+- :java-sync-api:`runCommand <com/mongodb/client/MongoDatabase.html#runCommand(org.bson.conversions.Bson)>`
+- :manual:`command <reference/command/>`
+- :manual:`read preference <core/read-preference/>`
```

Comment:
Nitpick: since you removed mention of the read preference, we should probably omit it here.